addons/main/script_version.hpp

```diff
@@ -1,4 +1,4 @@
-#define MAJOR 0
-#define MINOR 0
-#define PATCHLVL 0
```

Comment:
wait, why was that 0.0.0 ...what am I missing?

Reply:
`tools/update-versionfile.sh` updates this on release throught some black Dahlgren magic.

Reply:
![wtf](https://thumbs.gfycat.com/DisguisedFilthyKid-size_restricted.gif) 

hmm in that case we might want to remove script_version.hpp to avoid confusion